cubitpy/cubitpy.py

```diff
@@ -581,3 +581,10 @@ def display_in_cubit(self, labels=[], delay=0.5, testing=False):
             subprocess.call(cubit_command, cwd=cupy.temp_dir)
         else:
             return journal_path
+    
+    def import_fluent_geometry(self, file,feature_angle=135):
```

Comment:
Let's add this to the `cubitpy/cubit_utility.py` file as a free function (or maybe in a new file `geometry_import_functions.py`).

For context, the original idea behind the `CubitPy` object was to create a bare wrapper around the Cubit python interface and add BACI related functionality (which needs to track data, therefore this makes sense to implement in the same object) and the option to view the current state of the model. Over time a little more functionality was added to ease the selection of geometries. All other "extended functionality" should be implemented in free functions and not in the `CubitPy` class. This should make the update to newer Cubit versions easier. I know that some current methods in the class don't exactly follow that idea, we can think about moving them to free functions also. I am happy to discuss this in person!

Reply:
Yes, they also fall under that category - if it is not too much work, you can move them too, or I can move them in a follow up pull request.

Reply:
I think we should discuss this first and decide what should be moved and the structure. For some commands we are simply implementing the wrapped string for the command and I would actually not look in the utility folder first. So I would suggest identifying the functions which should be outsourced and deciding on the place (maybe `simple_wrapped_commands` ) in a different issue/discuss it shortly offline.

Reply:
Good idea, let's have this discussion, and then move the things around in separate PR.